Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -152,7 +152,7 @@ static void applyBackgroundColorForDebuggingToLayer(CALayer *layer, const WebCor
     }
 }
 
-static CALayer *createInteractionRegionLayer(WebCore::InteractionRegion::Type type, NSString *groupName)
+static CALayer *createInteractionRegionLayer(WebCore::InteractionRegion::Type type, WebCore::ElementIdentifier identifier, NSString *groupName)
```

Comment:
It's not great to use `ElementIdentifier` in the UI process, since, in a future world with site isolation, the same layer tree will contain layers from separate web processes with conflicting elementIDs.